APIs/SentinelHub/Overview/Authentication.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Authentication
+---
+
+The Sentinel Hub API uses OAuth2 Authentication and requires that you
+have an access token. In essence, this is a piece of information you add
+to your requests so the server knows it\'s you. These tokens do not last
+forever for a multitude of reasons, but you can get new ones and when
+they expire from the Sentinel-Hub OAuth2 server at the token endpoint
+listed below. But first, if you do not have one already, you need to
+register an OAuth Client in your [account
+settings](https://apps.sentinel-hub.com/dashboard/#/account/settings).
+This is so the server can expect you to make such token requests. If you
+do not have a Sentinel-Hub account, you can sign up for a [free trial
+account](https://www.sentinel-hub.com/create_account).
+
+#### How to use tokens
+
+Once you have a token, do use it for authenticating all your requests
+within its validity period. While tokens do not last forever, they do
+last a reasonable amount of time, and sufficiently long that they can be
+reused. The information of how long each token lasts is embedded in the
+token itself in the `exp` claim, and can be read from there.
+
+Do not fetch a new token for each API request you make. Token requests
+are rate limited, so if you are getting an HTTP 429 error, that means
+you are requesting too many tokens.
+
+Tokens are JSON Web Tokens (JWT), more information about them
+[here](https://en.wikipedia.org/wiki/JSON_Web_Token) or
+[here](https://datatracker.ietf.org/doc/html/rfc7519).
+
+### Registering OAuth client
+
+To register an OAuth client, open the \"User Settings\" tab in your
+dashboard, then click the `Create new` button (1) in the \"OAuth
+client\" section. Give your OAuth client a name (2), set the
+`Client grant type` to `Client Credentials`, and click the
+`Create client` button (3). Your client secret will be displayed. Copy
+the `secret` value (4) and paste it locally, as it will not be visible
+after the pop-up window closes! When you are finished, click `Close`
+(5). You should now see the newly created OAuth client name and ID (6)
+in the list of your OAuth clients. With client ID and client secret, you
+are now ready to request tokens.
+
+![](img/create_oauth_client.jpg)
```

Comment:
my suggestion would be to try and use captions for all images?

Reply:
On our side we don't use captions. On some images we do have title (onHover).